ducks/transactions.ts

```diff
@@ -209,19 +197,53 @@ export function processTransaction(
 
     try {
       const actionCall = actions[unconfirmedTx.actionName as keyof typeof actions]
-      const response = await (actionCall as any)(tinlake, ...unconfirmedTx.actionArgs)
-      hasCompleted = true
+      const tx = await (actionCall as any)(tinlake, ...unconfirmedTx.actionArgs)
 
-      const outcome = (response as any).status === SUCCESS_STATUS
-      console.log(outcome)
-      outcomeTx.status = outcome ? 'succeeded' : 'failed'
-      outcomeTx.result = response
+      if (tx.hash) {
+        // Confirmed
+        const pendingTx: Transaction = {
+          ...unconfirmedTx,
+          status: 'pending',
+          hash: tx.hash,
+        }
+        await dispatch({ id, transaction: pendingTx, dontChangeUpdatedAt: true, type: SET_ACTIVE_TRANSACTION })
+
+        const pendingTxTimeout = 10000
+        const hidePendingTxCallback = async () => {
+          if (hasCompleted) return
+
+          if (!document.hidden) {
```

Comment:
Nice!

Reply:
Yeah it doesn't work 100% perfectly to be honest, but I thought it was a neat quick fix ;-)